packages/concerto-util/lib/errorcodes.js

```diff
@@ -22,5 +22,7 @@ const DEFAULT_VALIDATOR_EXCEPTION = 'DefaultValidatorException';
 const REGEX_VALIDATOR_EXCEPTION = 'RegexValidatorException';
 // base exception for Type not found
 const TYPE_NOT_FOUND_EXCEPTION = 'TypeNotFoundException';
+// base exception for illegal model
+const Illegal_Model_Exception = 'IllegalModelException';
```

Comment:
Could you follow the capitalisation pattern already in place please?
```suggestion
const ILLEGAL_MODEL_EXCEPTION = 'IllegalModelException';
```